bootstrap.py

```diff
@@ -154,13 +135,14 @@ def get_vault(playbook):
     with open('/etc/ansible/hosts', 'a') as stream:
         stream.writelines(["\n[" + vault_name + "]\n", 'localhost\n'])
 
+
 def configure_environment():
```

Comment:
`detect()` feels more "magical" in this context. That might be okay, but could make debugging less intuitive. If `resource_tags()['Project']` is empty, it suggests an intuitive root cause (resource tag not set). If `detect('Project')` is empty, the context provides less suggestion about why that might be happening.